manual/src/README.md

```diff
@@ -0,0 +1,9 @@
+# web3_proxy manual
+
+Aliqua commodo eu sunt fugiat do amet fugiat non sunt labore ipsum Lorem. Esse et commodo ea reprehenderit dolore adipisicing aute Lorem. Occaecat excepteur aliqua proident nulla sunt commodo ex esse.
```

Comment:
Can you replace this with the text "Coming soon...." so that if people by chance get to the website they see that we are working on it? Same for the text on other md files.